src/Application.vala

```diff
@@ -184,6 +191,23 @@ public class Mail.Application : Gtk.Application {
 
         main_window.present ();
     }
+
+    private async bool request_background () {
+        var portal = new Xdp.Portal ();
+
+        Xdp.Parent? parent = active_window != null ? Xdp.parent_new_gtk (active_window) : null;
+
+        var command = new GenericArray<weak string> ();
+        command.add ("io.elementary.mail");
+        command.add ("--background");
+
+        try {
+            return yield portal.request_background (parent, _("Mail needs to run in background in order to send notifications."), (owned) command, Xdp.BackgroundFlags.AUTOSTART, null);
```

Comment:
this line pass the 120 line limit, would be good to break it.

```suggestion
            return yield portal.request_background (
                 active_window != null ? Xdp.parent_new_gtk (active_window) : null,
                _("Mail needs to run in background in order to send notifications."),
                (owned) command,
                Xdp.BackgroundFlags.AUTOSTART,
                null
            );
```

Also, IIRC, the background frontend do not check if the application already has permission, so this would end in the permission dialog begin shown in every launch even when the user already give permission.

in the calendar PR, it stores if it had an successful request and skip the request_background call in the next launches.